README.md

```diff
@@ -133,12 +131,12 @@ Data viewers/Editors based on shapes.
   - [DASH Data Shapes](https://www.datashapes.org/) - Platform-independent extensions of SHACL for common tasks. Stuff that could become an official standard in the future.
 
 - ShEx
-  - [Shape Expressions Language 2.1](https://shex.io/shex-semantics/index.html) - Final Community Group Report 8 October 2019.
-
-- ShEx, Drafts
-  - [P3330TM/D3 Draft Recommended Practice for Standard for Shape Expression Schemas](https://shexspec.github.io/spec/)
+  - [Shape Expressions Language 2.1](https://shex.io/shex-semantics/index.html) - Final Community Group Report: 8 October 2019
+  - [ShEx Primer](https://shexspec.github.io/primer/index.html) - Draft Community Group Report: 7 December 2022
+  - [ShEx Text Suite](https://github.com/shexSpec/shexTest) and [website](https://shexspec.github.io/shexTest/) - Version `next@1.0.2`: last release 5 October 2018
+  - [Recommended Practice for Standard for Shape Expression Schemas](https://shexspec.github.io/spec/) - IEEE Computer Society P3330/D3: Draft 9 October 2024. Next-version spec with `EXTENDS` keyword
+  - [ShEx Primer](https://shex.io/primer-next/) - Primer with `EXTENDS` keyword: Draft Community Group Report 25 May 2022
```

Comment:
These colons are inappropriate for these placements. I don't care if programmers dislike commas. They're sometimes the appropriate punctuation.